plugins/languages/treesitter/treesitter.nix

```diff
@@ -118,7 +118,11 @@ helpers.neovim-plugin.mkNeovimPlugin config {
 
       keymaps =
         let
-          mkKeymap = default: helpers.defaultNullOpts.mkStr default "Key shortcut";
+          mkKeymap =
+            default:
+            helpers.defaultNullOpts.mkNullable (
```

Comment:
```suggestion
            helpers.defaultNullOpts.mkNullableWithRaw (
```

Reply:
Or, since it'd be nested `either`s otherwise, we could use `oneOf` here:

```nix
helpers.defaultNullOpts.mkNullable (
  with helpers.nixvimTypes;
  oneOf [
    str
    bool
    rawLua
  ]
) default "Key shortcut or false to unset.";
```